bin/install-plugin.sh

```diff
@@ -20,38 +20,50 @@
 #All are downloaded by default. You can also choose what you need. 
 #You only need to configure the plug-in name in config/plugin_config.
 
+# get seatunnel home
+SEATUNNEL_HOME=$(cd $(dirname $0);cd ../;pwd)
+
 # connector default version is 2.2.0, you can also choose a custom version. eg: 2.1.2:  sh install-plugin.sh 2.1.2
```

Comment:
Perhaps the easiest way to determine whether the current `bin` directory or the `root` directory, if it's the `bin` directory, then `cd..` Since we use maven to download the jar, the`maven-wrapper` file must correspond to the directory.